charts/oes/templates/pipeline-promotion/local-spin-cli-config-secret.yaml

```diff
@@ -9,5 +9,5 @@ stringData:
 kind: Secret
 metadata:
   name: local-spin-cli-config
-  namespace: {{ .Values.spinnaker.gitopsHalyard.pipelinePromotion.executionNamespace }}
+  #namespace: {{ .Values.spinnaker.gitopsHalyard.pipelinePromotion.executionNamespace }}
```

Comment:
Let's remove this namespace section and let pipeline promotion stuff run in the same namespace as OES. Would it have any side effects?

Reply:
Will remove comments after testing.